src/web/theme/ui/organisms/Media/Media.scss

```diff
@@ -1,42 +1,94 @@
-.media {
+.media-container {
   position: relative;
-  background: $shade02;
-  overflow: hidden;
+  background: none;
   width: 100%;
+  &:before,
+  &:after {
+    content: "";
+    position: absolute;
+    width: 1px;
+    height: $mediaSpacing;
+    background: $shade02;
+    left: 50%;
+  }
+  &:before {
+    top: -$mediaSpacing;
+  }
+  &:after {
+    bottom: -$mediaSpacing;
+  }
 
-  img {
+  .media {
+    position: relative;
+    background: none;
+    overflow: hidden;
     width: 100%;
     height: 100%;
-    max-height: 100vh;
-    object-fit: cover;
-    object-position: center;
-  }
+    img {
+      width: 100%;
+      height: 100%;
+      max-height: 100vh;
+      object-fit: cover;
+      object-position: center;
+    }
 
-  &__details {
-    cursor: pointer;
-    position: absolute;
-    bottom: 0;
-    left: 0;
-    width: 100%;
-    max-height: 100%;
-    overflow-y: auto;
-    background: $brandPrimaryBackground;
-    color: $reverseFontColor;
-    border: 0;
-    padding: 1em;
-    box-sizing: border-box;
+    &__details {
+      cursor: pointer;
+      position: absolute;
+      bottom: 0;
+      right: 0;
+      width: 50%;
+      max-height: 100%;
+      overflow-y: auto;
+      color: $reverseFontColor;
+      border: 0;
+      box-sizing: border-box;
 
-    opacity: 0;
-    transform: translate(0, 50%);
-    transition: opacity 0.3s ease, transform 0.3s ease-in;
-  }
+      opacity: 0;
+      transform: translate(0, 50%);
+      transition: opacity 0.3s ease, transform 0.3s ease-in;
+
+      display: flex;
+      flex-wrap: wrap;
+
+      span,
+      a,
+      button {
+        flex: 1;
+        color: $backgroundColor;
+      }
+      &__title {
+        flex: 1 1 100%;
+        padding: 0.5em;
+        padding-left: 2em;
+        background: $backgroundColor;
+        opacity: 0.75;
+        color: $fontColor;
+        font-family: $mainFontFamily;
+        font-size: $h1;
+        font-weight: $bold;
+        text-align: right;
+      }
+    }
+
+    &.product-item-title {
```

Comment:
I think this component declaration should be extracted in its own component, with its BEM namespace… and more expressive naming conventions.
It does not really belong here, and the current implementation has room for improvements.